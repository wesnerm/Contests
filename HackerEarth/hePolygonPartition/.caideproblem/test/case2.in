20
800 341
597 162
772 717
679 948
207 836
110 247
937 640
78 414
24 999
490 37
908 147
908 349
816 178
193 693
830 931
292 746
248 159
743 683
245 498
244 783