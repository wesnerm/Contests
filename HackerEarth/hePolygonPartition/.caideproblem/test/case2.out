5
4 9 6 11 15
4 8 10 7 5
4 20 17 13 4
4 14 2 12 3
4 19 1 18 16
